src/components/MetaData/index.test.tsx

```diff
@@ -18,1214 +18,1230 @@ jest.mock('react-dom/server', () => ({
+  expect(wrapper.find("meta[name='twitter:card']").props().content).toBe('summary_large_image');
 };
 
 const expectTwitterMetaMissing = (wrapper: ShallowWrapper): void => {
-  expect(wrapper.find("meta[property='twitter:site']").length).toBe(0);
-  expect(wrapper.find("meta[property='twitter:card']").props().content).toBe('summary_large_image');
+  expect(wrapper.find("meta[name='twitter:site']").length).toBe(0);
+  expect(wrapper.find("meta[name='twitter:card']").props().content).toBe('summary_large_image');
 };
 
-describe('the meta data ', () => {
-  describe('if page type is article', () => {
-    describe('when globalContent is complete', () => {
-      const metaValue = (prop: string): string | null => {
-        if (prop === 'page-type') {
-          return 'article';
-        }
-        if (prop === 'title') {
-          return 'the-sun';
-        }
-        return null;
-      };
-
-      const useFusionContext = {
-        globalContent: {
-          description: {
-            basic: 'this is a description',
-          },
-          headlines: {
-            basic: 'this is a headline',
-          },
-          taxonomy: {
-            // eslint-disable-next-line @typescript-eslint/camelcase
-            seo_keywords: [
-              'keyword1',
-              'keyword2',
-            ],
-            tags: [
-              { slug: 'tag1' },
-              { slug: 'tag2' },
-            ],
-          },
-          // eslint-disable-next-line @typescript-eslint/camelcase
-          promo_items: {
-            basic: {
-              url: 'awesome-url',
-              // eslint-disable-next-line @typescript-eslint/camelcase
-              alt_text: 'alt text',
-            },
-          },
-        },
-        arcSite: 'the-sun',
-      };
-      const { globalContent } = useFusionContext;
-      const wrapper = shallow(<MetaData
-        metaValue={metaValue}
-        MetaTag={jest.fn()}
-        MetaTags={jest.fn()}
-        globalContent={globalContent}
-        twitterUsername={twitterUsername}
-        websiteName={websiteName}
-        resizerURL={resizerURL}
-      />);
+const wrapperGenerator = (
+  metaValue: MetaValuesReturnInterface,
+  globalContent: GlobalContentBag,
+): ShallowWrapper => (
+  shallow(
+    <MetaData
+      metaValue={metaValue}
+      MetaTag={jest.fn()}
+      MetaTags={jest.fn()}
+      globalContent={globalContent}
+      twitterUsername={twitterUsername}
+      websiteName={websiteName}
+      resizerURL={resizerURL}
+      arcSite={arcSite}
+      websiteDomain={websiteDomain}
+      facebookAdmins={facebookAdmins}
+    />,
+  )
+);
+
+const metaValues = (values: object): MetaValuesReturnInterface => {
+  const data = { ...values };
+  return (prop: string): string | null => data[prop];
+};
 
-      it('should have a title', () => {
-        expect(wrapper.find('title').length).toBe(1);
+const titleTest = (pageType: string): void => {
+  describe('when need to add a title tag', () => {
```

Comment:
is there a test for when global content is/isn't provided?

Reply:
yes, each page type has a section to test for globalContent missing, like this
https://github.com/WPMedia/engine-theme-sdk/blob/0e7f7fdc5ad2502b179d929a6583ffad9350a1e5/src/components/MetaData/index.test.tsx#L560